src/appkit.cs

```diff
@@ -1174,13 +1201,16 @@ interface NSBezierPath : NSCoding, NSCopying {
 		[Export ("appendBezierPathWithArcFromPoint:toPoint:radius:")]
 		void AppendPathWithArc (CGPoint point1, CGPoint point2, nfloat radius);
 
+		[Obsolete ("Use 'AppendBezierPathWithCGGlyph' instead.")]
```

Comment:
This is actually the NSGlyph version obsoleted by the CGGlyph. It should have been Availability (Obsoleted) not Obsolete unlike the other renames.

Fixing (and we don't appear to be ```#if XAMCORE_4_0``` those.